.eslintrc.js

```diff
@@ -420,6 +420,23 @@ module.exports = {
 			plugins: [ 'ssr-friendly' ],
 			extends: [ 'plugin:ssr-friendly/recommended' ],
 		},
+		{
+			files: [ 'packages/components/src/**' ],
+			rules: {
+				'no-restricted-imports': [
+					'error',
+					{
+						paths: restrictedImports.filter(
+							( { name } ) =>
+								! [
+									'@ariakit/react',
+									'framer-motion',
+								].includes( name )
+						),
+					},
+				],
+			},
+		},
```

Comment:
It may be useful to add a comment that we're re-exporting them in the package so we're allowing usage inside the package. Right now it's not so obvious without looking deeper into why they were forbidden in the first place.

Reply:
Sure, I'll add a comment